### ISSUE TITLE
Installation error on development_stabilize

### ISSUE THREAD

**Issue by serboctor** [threefoldtech/jumpscaleX_core#78]
In system installation using `jsx install` works fine. However, container installation using `jsx container-install` gives the following error even when the ssh key is actually loaded.
```
Traceback (most recent call last):
  File "jsx", line 587, in <module>
    cli()
  File "/usr/lib/python3/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "jsx", line 294, in install
    _configure(configdir="/sandbox/cfg", basedir="/sandbox", no_interactive=no_interactive)
  File "jsx", line 96, in _configure
    configdir=configdir,
  File "/tmp/InstallTools.py", line 3419, in configure
    self.sshagent.key_default_name
  File "/tmp/InstallTools.py", line 4909, in key_default_name
    raise Tools.exceptions.Base("sshkey should be passed forward by means of SSHAgent")
IT.BaseJSException: null
````
I think the installer doesn't forward the key correctly.

**Comment by hossnys** [comment]
just tested container install now on latest changes on development_stabilize branch and it's working fine except it show this after installation 
```
        install succesfull:

        # if you use a container do:
        jsx container-kosmos

        or

        kosmos

```

which need to be ```/tmp/jsx container-kosmos```

**Comment by hossnys** [comment]
fixed here c5b9cf3 and verified .
